src/overlays/actors/ovl_Boss_Ganondrof/z_boss_ganondrof.c

```diff
@@ -1,15 +1,97 @@
+/*
+ * File: z_boss_ganondrof.c
+ * Overlay: ovl_Boss_Ganondrof
+ * Description: Phantom Ganon
+ */
+
 #include "z_boss_ganondrof.h"
+#include "objects/object_gnd/object_gnd.h"
+#include "overlays/actors/ovl_En_fHG/z_en_fhg.h"
+#include "overlays/actors/ovl_En_Fhg_Fire/z_en_fhg_fire.h"
+#include "overlays/effects/ovl_Effect_Ss_Fhg_Flash/z_eff_ss_fhg_flash.h"
+#include "overlays/effects/ovl_Effect_Ss_Hahen/z_eff_ss_hahen.h"
 
 #define FLAGS 0x00000035
 
 #define THIS ((BossGanondrof*)thisx)
 
+typedef enum {
+    /* 0 */ NOT_DEAD,
+    /* 1 */ DEATH_START,
+    /* 2 */ DEATH_THROES,
+    /* 3 */ DEATH_WARP,
+    /* 4 */ DEATH_SCREAM,
+    /* 5 */ DEATH_DISINTEGRATE,
+    /* 6 */ DEATH_FINISH
+} BossGanondrofDeathStates;
```

Comment:
Relevant to all the enums in this file
```suggestion
} BossGanondrofDeathState;
```